### ISSUE TITLE
E38 Support: Find out what the 0x04 and 0x03 chars print out to

### ISSUE THREAD

**Issue by tedsalmon** [t3ddftw/DroidIBus#48]
```
07-20 23:53:59.474 D/DroidIBus(2613): Received Message (Radio -> GraphicsNavigationDriver): 68 12 3B 23 40 20 46 4D 20 03 20 38 38 2E 35 04 20 20 20 35 
07-20 23:53:59.474 D/DroidIBus(2613): Triggering 'onUpdateRadioStation()' with value 'FM EXT 88.5 EOT'
07-20 23:53:59.474 D/DroidIBus(2613): Triggering 'onUpdateRadioStation()' with value 'FM EXT 88.5 EOT'
```

**Comment by tedsalmon** [comment]
They don't print out so filter them out.

**Comment by tedsalmon** [comment]
Fixed in #59
